test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestPackageUtility.cs

```diff
@@ -237,11 +237,8 @@ public static void CreatePackage(Stream stream, SimpleTestPackageContext package
             {
                 using (var signPackage = new SignedPackageArchive(tempStream, stream))
                 {
-                    var testSignatureProvider = new X509SignatureProvider(timestampProvider: null);
-                    var signer = new Signer(signPackage, testSignatureProvider);
-                    var request = new AuthorSignPackageRequest(packageContext.AuthorSignatureCertificate, hashAlgorithm: HashAlgorithmName.SHA256);
-
-                    signer.SignAsync(request, testLogger, CancellationToken.None).Wait();
+                    var request = new AuthorSignPackageRequest(packageContext.AuthorSignatureCertificate, HashAlgorithmName.SHA256);
+                    AddSignatureToPackageAsync(signPackage, request, testLogger).Wait();
```

Comment:
Change the method to async and drop the `Wait()`.

Also, wrap `request` in a `using` statement.

Reply:
@dtivel I didn't want to drop the Wait() because this method is used all over our tests, therefore making it async would require to change a bunch of our current test code. I would prefer doing that as part of another PR if needed.